scripts/config.js

```diff
@@ -1,5 +1,4 @@
-
-exports.linkDistance = 500; //meters, whole thing is about 800 km 
-exports.startLat = -47.28342; //  47°17'0.31"S
+exports.linkDistance = 500 // meters, whole thing is about 800 km
+exports.startLat = -47.28342 // 47°17'0.31"S
 exports.startLon = 167.50099 // 167°30'3.56"E
-exports.brng = 40.11
\ No newline at end of file
+exports.brng = 40.11 // bearing
```

Comment:
This really should just be `exports.bearing = 40.11`.
Actually, if it were me, I would:
```
exports = {
  linkDistance: 500,
  startLat: -47.28342,
  startLon: 167.50099,
  bearing: 40.11
}
```
(would require a couple of code changes on code that uses this)